notebooks/env.ipynb

```diff
@@ -0,0 +1,692 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": [],
+      "collapsed_sections": [],
+      "toc_visible": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    },
+    "accelerator": "GPU"
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1n3uLs7P5wg0yLsgaa0ipfudQ_QwknT5v?usp=sharing)"
+      ],
+      "metadata": {
+        "id": "P6rofPFBkF8B"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# About the tutorial\n",
+        "Datasets are essential in both supervised and unsupervised machine learning settings. In a typical reinforcement learning (RL) setting, the agent must interact with the environment in order to collect data for learning. Thus, environments serve a kind of similar function in RL as datasets do in supervised and unsupervised learning. In this tutorial, we will explain how to use RLHive environments. Note that this tutorial is on single-agent environments."
+      ],
+      "metadata": {
+        "id": "TiiOCUnwDlXB"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "qy1F1g74qfo_"
+      },
+      "source": [
+        "# Introduction and Setup"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### What is RLHive and how to install it"
```

Comment:
Why is there a section like this one in the `env.py`? I think the users would probably know what the RLHive is by the time they want to use the environments. If you agree, change it, please.